draft-tigress-general-requirements.md

```diff
@@ -43,59 +43,62 @@ author:
-    date: 2022-11
-    target: https://datatracker.ietf.org/doc/draft-art-tigress/
 
+
+informative:
   CCC-Digital-Key-30:
     author:
       org: Car Connectivity Consortium
     title: "Digital Key Release 3"
     date: 2022-07
     target: https://carconnectivity.org/download-digital-key-3-specification/
 
-informative:
-
 
 --- abstract
 
 
-This document describes the use cases necessitating the secure transfer of digital credentials, residing in a digital wallet, between two devices and defines general assumptions, requirements and the scope of the corresponding Tigress Internet-draft {{Tigress-00}}.
+This document describes the use cases necessitating the secure transfer of digital credentials, residing in a digital wallet, between two devices and defines general assumptions, requirements and the scope for possible solutions to this problem.
 
 --- middle
 
 # Introduction
 
+In this document we are identifying a problem of transferring digital credentials (e.g. a digital car key, a digital key a a hotel room or a digital key to a private house) from a wallet on one device (smartphone) to another, particularly, if these devices belong to two different platforms (e.g. one is iOS, another - Android).
 Today, there is no widely accepted way of transferring digital credentials securely between two digital wallets independent of hardware and software manufacturer. This document describes the problem space and the requirements for the solution the working group creates.
 
-Tigress allows for a sender and receiver to communicate in order to facilitate a secure credential transfer between two digital wallets. Tigress also specifies certain privacy requirements in order to maintain a high level of user privacy.
+A Working Group, called Tigress has been established to find a solution to the problem described above.
+Within the WG an initial solution was presented (https://datatracker.ietf.org/doc/draft-art-tigress). The community decided to generalize the requirements to the solution and consider alternative solutions within the WG.
+
+This document presents the general requirements to possible solutions and  specifies certain privacy requirements in order to maintain a high level of user privacy.
 
 # General Setting
 
-When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two digital wallets, potentially through an intermediary server, there are a couple more actors involved.
+When sharing digital secure credentials, there are several actors involved. This document will focus on sharing information between two digital wallets, directly or through an intermediary server.
+
+Digital credentials provide access to property owned and operated by 3-rd party entities, such as hotel or residential building owners. The entity that is providing the digital credential for consumption by a digital wallet are referred to as Provisioning Partners.
```

Comment:
Should this be just operated by 3rd party entities in the case of car key?

Reply:
changed to "owned and / or operated"